composer.json

```diff
@@ -19,46 +19,67 @@
             "typo3/cms-composer-installers": true,
             "typo3/class-alias-loader": true
         },
+        "bin-dir": ".Build/bin",
+        "sort-packages": true,
         "vendor-dir": ".Build/vendor"
     },
     "require-dev": {
         "t3docs/codesnippet": "dev-main",
-        "typo3/cms-adminpanel": "dev-main as 12.0",
-        "typo3/cms-backend": "dev-main as 12.0",
-        "typo3/cms-belog": "dev-main as 12.0",
-        "typo3/cms-beuser": "dev-main as 12.0",
-        "typo3/cms-dashboard": "dev-main as 12.0",
-        "typo3/cms-extbase": "dev-main as 12.0",
-        "typo3/cms-extensionmanager": "dev-main as 12.0",
-        "typo3/cms-felogin": "dev-main as 12.0",
-        "typo3/cms-filelist": "dev-main as 12.0",
-        "typo3/cms-filemetadata": "dev-main as 12.0",
-        "typo3/cms-fluid": "dev-main as 12.0",
-        "typo3/cms-fluid-styled-content": "dev-main as 12.0",
-        "typo3/cms-form": "dev-main as 12.0",
-        "typo3/cms-frontend": "dev-main as 12.0",
-        "typo3/cms-impexp": "dev-main as 12.0",
-        "typo3/cms-indexed-search": "dev-main as 12.0",
-        "typo3/cms-info": "dev-main as 12.0",
-        "typo3/cms-install": "dev-main as 12.0",
-        "typo3/cms-linkvalidator": "dev-main as 12.0",
-        "typo3/cms-lowlevel": "dev-main as 12.0",
-        "typo3/cms-opendocs": "dev-main as 12.0",
-        "typo3/cms-recycler": "dev-main as 12.0",
-        "typo3/cms-redirects": "dev-main as 12.0",
-        "typo3/cms-reports": "dev-main as 12.0",
-        "typo3/cms-rte-ckeditor": "dev-main as 12.0",
-        "typo3/cms-scheduler": "dev-main as 12.0",
-        "typo3/cms-seo": "dev-main as 12.0",
-        "typo3/cms-setup": "dev-main as 12.0",
-        "typo3/cms-sys-note": "dev-main as 12.0",
-        "typo3/cms-t3editor": "dev-main as 12.0",
-        "typo3/cms-tstemplate": "dev-main as 12.0",
-        "typo3/cms-viewpage": "dev-main as 12.0",
-        "typo3/cms-workspaces": "dev-main as 12.0",
-        "t3docs/examples": "dev-main"
+        "typo3/cms-adminpanel": "^12.4 | dev-main",
```

Comment:
Sure you want to use `^12.4` on main branch and `dev-main` on 12.4 branch?

Reply:
what would you suggest? only dev-main for main and only ^12.4 for 12.4?